docs/cli.md

```diff
@@ -32,6 +32,8 @@ you can run the following command from your `openwhisk` directory:
 ./bin/wsk property set --auth `cat ansible/files/auth.guest`
 ```
 
+**Tip:** The OpenWhisk CLI stores the properties set in `~/.wskprops` by default. The location of this file can be altered by setting the `WSK_CONFIG_FILE` environment variable.
+
```

Comment:
Do you mind adding one more line to close https://github.com/openwhisk/openwhisk/issues/1908 as well?

If you are using an local OpenWhisk deployment with a self-signed SSL certificate, you can use `--insecure` to bypass certificate validation.